azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/bindings.json

```diff
@@ -91,6 +91,37 @@
         }
       ]
     },
+    {
+      "type": "blobTrigger",
```

Comment:
please change `type` to `EventGridBlobTrigger` to join with the `templates.json`

Reply:
Updated `type` name